lib/gateway/Shard.js

```diff
@@ -594,6 +594,23 @@ class Shard extends EventEmitter {
                 });
                 break;
             }
+            case "MESSAGE_REACTION_REMOVE_EMOJI": {
+                const channel = this.client.getChannel(packet.d.channel_id);
+                if(!channel) {
+                    break;
+                }
+                /**
+                * Fired when someone removes all reactions from a message for a single emoji
+                * @event Client#messageReactionRemoveAllForEmoji
+                * @prop {Message | Object} message The message object. If the message is not cached, this will be an object with `id` and `channel` keys. No other property is guaranteed
+                * @prop {Object} emoji The emoji object with a `name` prop. If the emoji is a custom emoji it will also have an `id` prop.
+                */
+                this.emit("messageReactionRemoveAllForEmoji", channel.messages.get(packet.d.message_id) || {
```

Comment:
Event name should probably be `messageReactionRemoveEmoji`. Not as wordy, and it matches the payload event name